<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>635</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>321</width>
      <height>411</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Covariates</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>301</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Configuration</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Whiskers</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>251</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Frame</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Whisker 1</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="TimeScrollBar" name="time_scrollbar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>509</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Video"/>
    <addaction name="actionLoad_Images"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionTake_Snapshot"/>
   </widget>
   <widget class="QMenu" name="menuModules">
    <property name="title">
     <string>Modules</string>
    </property>
    <addaction name="actionWhisker_Tracking"/>
    <addaction name="actionLabel_Maker"/>
    <addaction name="actionAnalog_Viewer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuImage"/>
   <addaction name="menuModules"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Video">
   <property name="text">
    <string>Load Video</string>
   </property>
  </action>
  <action name="actionTake_Snapshot">
   <property name="text">
    <string>Take Snapshot</string>
   </property>
  </action>
  <action name="actionWhisker_Tracking">
   <property name="text">
    <string>Whisker Tracking</string>
   </property>
  </action>
  <action name="actionLabel_Maker">
   <property name="text">
    <string>Label Maker</string>
   </property>
  </action>
  <action name="actionLoad_Images">
   <property name="text">
    <string>Load Images</string>
   </property>
  </action>
  <action name="actionAnalog_Viewer">
   <property name="text">
    <string>Analog Viewer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeScrollBar</class>
   <extends>QWidget</extends>
   <header>TimeScrollBar/TimeScrollBar.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
